tests/vars/main.yml

```diff
@@ -2,3 +2,14 @@
 ---
 rstudio_connect_install:
   - r-base
+rstudio_connect_config:
+  R:
+    ExecutableSystemScanning: true
```

Comment:
RStudio introduced a new parameters for [scanning R binaries](https://docs.rstudio.com/connect/admin/r/#scanning). 
By default `ExecutableSystemScanning` is set to `false` - so Connect will not look for R which is installed via (system) package manager. This broke molecule tests in the role - I updated variables for tests, so they pass now